salt/states/pip_state.py

```diff
@@ -186,16 +186,19 @@ def _check_pkg_version_format(pkg):
 
 def _check_if_installed(prefix, state_pkg_name, version_spec, ignore_installed,
                         force_reinstall, upgrade, user, cwd, bin_env, env_vars,
-                        **kwargs):
+                        pip_list=False, **kwargs):
```

Comment:
IMHO we need to have some documentation about what this kwarg does and the type of data expected here.

Reply:
I was surprised by that kwargs too! It wasn't in the release code I initially patched on my systems, but it was there when I pulled the 2017.7 branch to do the pull request against.

Looks like it came from here: https://github.com/saltstack/salt/commit/7c46d9d0d457a686afb12860a655c0ecc15e427b

Reply:
@OrlandoArcapix I think we just want some documentation for your new kwarg, `pip_list`. I know the terminology was a little confusing there. :)